polling_stations/apps/data_collection/base_importers.py

```diff
@@ -657,9 +657,12 @@ def get_station_point(self, record):
                 srid=27700)
         else:
             # otherwise, geocode using postcode
+            postcode = getattr(record, self.station_postcode_field).strip()
+            if postcode == '':
```

Comment:
If we did make it this far and this value was either `None` or `0` then the previous line just threw
`AttributeError: 'NoneType' object has no attribute 'strip'` or
`AttributeError: 'int' object has no attribute 'strip'`
so this check would not be reached.

In general the code that processes stuff sourced from `CsvHelper` assumes we are working with strings because `csv.reader` returns each row as a list of strings.